static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -3,18 +3,48 @@ define([
     'fastdom',
     'common/modules/navigation/edition-picker',
     'common/modules/navigation/editionalise-menu'
-], function (
-    qwery,
-    fastdom,
-    editionPicker,
-    editionaliseMenu
-) {
+], function (qwery,
```

Comment:
IntelliJ strikes again! This indentation should be fixed to be the same as the other files in our codebase